src/main/webapp/main/Routes.js

```diff
@@ -17,33 +17,36 @@ import DataUseRestrictionEdit from '../dataUse/DataUseRestrictionEdit';
-    <Route path= {"/project/main"} render = {(routeProps) => <Main {...routeProps} {...props}/> }/>
-    <Route path= {"/newConsentGroup/main"} render = {(routeProps) => <Main {...routeProps} {...props}/> }/>
-    <Route path= {"/admin/fundingReport"} render = {(routeProps)=> <FundingsSourceReport {...routeProps} {...props}/>}/>
-    <Route path= {"/report/reviewCategories"} render = {(routeProps) => <ReviewCategories {...routeProps} {...props}/> }/>
-    <Route path= {"/statusEvent/qaEventReport"} render = {(routeProps) => <QaReport {...routeProps} {...props}/>} />
-    <Route path= {"/statusEvent/projectReport"} render = {(routeProps) => <ProjectReport {...routeProps} {...props}/>} />
-    <Route path= {"/index/profile"} render = {(routeProps) =>  <Profile {...routeProps} {...props}/> }/>
-    <Route path= {"/index/about"} render = {(routeProps) =>  <AboutPage {...routeProps} {...props}/> }/>
-    <Route path= {"/profile"} render = {(routeProps) =>  <Profile {...routeProps} {...props}/> }/>
+    <AuthenticatedRoute path= {"/user/rolesManagement"} component={RolesManagement} props={props} admin={true}/> }/>
+    <AuthenticatedRoute path= {["/project/main", "/newConsentGroup/main"]} component={Main} props={props}/> }/>
+    <AuthenticatedRoute path= {"/project/pages"} component={NewProject} props={props}/> }/>
+    <AuthenticatedRoute path= {"/admin/fundingReport"} component={FundingsSourceReport} props={props} admin={true}/> }/>
+    <AuthenticatedRoute path= {"/report/reviewCategories"} component={ReviewCategories} props={props} admin={true}/> }/>
+    <AuthenticatedRoute path= {"/statusEvent/qaEventReport"} component={QaReport} props={props} admin={true}/> }/>
+    <AuthenticatedRoute path= {"/statusEvent/projectReport"} component={ProjectReport} props={props} admin={true} />
+    <AuthenticatedRoute path= {"/dataUse/list"}  component={DataUseRestrictionIndex} props={props} admin={true}/> }/>
+    <AuthenticatedRoute path= {"/dataUse/view"}  component={DataUseRestrictionDetails} props={props}/> }/>
+    <AuthenticatedRoute path= {"/consent-group/use-existing"} component={LinkWizard} props={props}/> }/>
+    <AuthenticatedRoute path= {"/consent-group/new"} component={NewConsentGroup} props={props}/> }/>
+    <AuthenticatedRoute path= {"/dataUse/restriction"} component={DataUseRestrictionEdit} props={props}/> }/>
+    <AuthenticatedRoute path= {"/issueList/list"} component={IssueList} props={props} /> }/>
+    <AuthenticatedRoute path= {"/search/index"} component={Search} props={props} /> }/>
+    <AuthenticatedRoute path= {"/submissions/add-new"} component={SubmissionForm} props={props} /> }/>
+    <Route path= {"/infoLink/showInfoLink"} component={InfoLink} props={props}/> }/>
+    <Route path= {["/index", "/"]} exact component={LandingPage} props={props}/> }/>
+    <Route path= {["/index/profile", "/profile"]} render = {(routeProps) =>  <Profile {...routeProps} {...props}/> }/>
```

Comment:
`/infoLink/showInfoLink` and `/index/profile` should be wrapped in AuthenthicatedRoute.